code/src/components/Survey.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { StartDay } from './StartDay';
+import { Relationships } from './Relationships';
+import { WhatKinde } from './WhatKinde';
+import { Feeling } from './Feeling';
+
+import { Buttons } from './Buttons';
+
+export const Survey = (props) => {
+  return (
+    <form className="form">
```

Comment:
Maybe I'm understanding this wrong, but seems like you're doing it the way I did it. you created a component to wrap all questions and then print it on App.js. I also did the same with this part.